resources/src/main/java/org/robolectric/res/android/ResourceTypes.java

```diff
@@ -1604,5 +1604,5 @@ protected void put(int position, Short value) {
     }
   }
 
-  private static final Runnable NO_OP = () -> {};
```

Comment:
Is this even being used? Can it be deleted?

Reply:
Yes, this was not being used anywhere

Reply:
What about deleting it?

Reply:
@utzcoz Sure 👍🏻